src/test/ui/issues/issue-49934.rs

```diff
@@ -1,11 +1,13 @@
-// build-pass (FIXME(62277): could be check-pass?)
```

Comment:
Can't get around this since `#[derive(Debug)` now throws an error

Reply:
Could you split the test into two tests - one with errors and one with warnings?

Reply:
Done